src/components/Menu.vue

```diff
@@ -28,7 +28,7 @@
           <ion-label class="ion-text-wrap">
             <p class="overline">{{ instanceUrl }}</p>
             {{ eComStore.storeName }}
-            <p>{{ currentShopifyConfig.name ? currentShopifyConfig.name : currentShopifyConfig.shopifyConfigName }}</p>
+            <p>{{ currentShopifyConfig?.name ? currentShopifyConfig.name : currentShopifyConfig?.shopifyConfigName }}</p>
```

Comment:
I think we should set it as empty object by default and this check will not be required then

Reply:
Set empty object by default to resolve this issue